logic.js

```diff
@@ -0,0 +1,75 @@
+const Tweeter = function () {
+    const posts = [
+        {
+            text: "First post!",
+            id: "p1",
+            comments: [
+                { id: "c1", text: "First comment on first post!" },
+                { id: "c2", text: "Second comment on first post!!" },
+                { id: "c3", text: "Third comment on first post!!!" }
+            ]
+        },
+        {
+            text: "Aw man, I wanted to be first",
+            id: "p2",
+            comments: [
+                { id: "c4", text: "Don't wory second poster, you'll be first one day." },
+                { id: "c5", text: "Yeah, believe in yourself!" },
+                { id: "c6", text: "Haha second place what a joke." }
+            ]
+        }
+    ]
+    let postIdCounter = 3
+    let commentIdCounter = 7
+
+    const getPosts = function () {
+        return posts
+    }
+    const removePost = function(postId){
+        for (let j in posts){
+            if(posts[j].id ===postId){
+                posts.splice(j,1)
+            }
+        }
+    }
+
+    const addPost = function (text) {
+        const newPost = {
+            text: text,
+            id: "p" + postIdCounter,
+            comments: []
+        }
+        postIdCounter++
```

Comment:
Change the order of these lines just so it's easier to read. Create the object, push it, then increment the counter.